### PR TITLE
feature: Make usable without atty.

### PR DESCRIPTION
**Pull request by shanecelis**
I'm thinking about using this crate with bevy for doing colored text without any tty in sight. This merely adds a feature 'tty' that is by default on, so no change unless opted-in.

### CODE REVIEW

File: src/control.rs [hunk 1]
```diff
@@ -103,9 +103,16 @@ impl ShouldColorize {
     /// `CLICOLOR_FORCE` takes highest priority, followed by `NO_COLOR`,
     /// followed by `CLICOLOR` combined with tty check.
     pub fn from_env() -> Self {
+        #[allow(unused_mut)]
+        let mut tty: Option<bool> = None;
+
+        #[cfg(feature = "tty")]
+        {
+            tty = Some(atty::is(atty::Stream::Stdout));
+        }
```
Comment (hwittenborn) [inline src/control.rs:112]: The `Option` could just be removed, right?

```rust
        #[allow(unused_mut)]
        let mut tty = false;

        #[cfg(feature = "tty")]
        {
            tty = atty::is(atty::Stream::Stdout);
        }
```

**Review by hwittenborn — CHANGES_REQUESTED** [review]

**Comment by hwittenborn** [comment]
Ignore the CI failing, that's due to all the features not being enabled. I'll get that fix shortly.

**Comment by hwittenborn** [comment]
Hey, @shanecelis, I've gone ahead and replaced `atty` with `is-terminal` in the latest release because of #131. If you could get this PR moved over to that I'd still be more than glad to accept it :)

**Comment by shanecelis** [comment]
Great. Will do.

**Pull request closed without merging**